test/api/test_invoice.py

```diff
@@ -204,6 +204,55 @@
 }
 """
 
+retrieveInvoiceExample = """
+{
+  "uuid": "inv_123",
```

Comment:
Well, the UUID doesn't parse anywhere in this library & this is just a mocked test.
It's testing that the overall method works and it's parsable into a python object.
Also, we don't validate UUIDs in client libraries, that is done on server side. Here, it's just a string.
I don't want to duplicate unnecessarily.